src/libraries/System.DirectoryServices/ref/System.DirectoryServices.csproj

```diff
@@ -10,7 +10,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="$(LibrariesProjectRoot)System.Security.Permissions\ref\System.Security.Permissions.csproj" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Security.Permissions\ref\System.Security.Permissions.csproj" PrivateAssets="all" />
```

Comment:
nit: consider adding the same comment here as in src.

Reply:
Thanks. Done